### PR TITLE
Update cancel and confirm transaction function after MM version updated

### PR DESCRIPTION
**Pull request by jake4take**
fix: rework the cancel and confirm transaction function

### CODE REVIEW

File: packages/wallets/src/metamask/pages/walletOperations.page.ts [hunk 1]
```diff
@@ -22,17 +22,17 @@ export class WalletOperationPage {
     this.page = page;
     this.nextButton = this.page.getByTestId('page-container-footer-next');
     this.confirmButton = this.page.getByTestId('confirm-footer-button');
-    this.rejectButton = this.page.getByTestId('page-container-footer-cancel');
+    this.approvalRejectButton = this.page.getByTestId(
```
Comment (itaven) [inline packages/wallets/src/metamask/pages/walletOperations.page.ts:25]: Does "reject" wording applies only for approval modal?

File: packages/wallets/src/metamask/metamask.page.ts [hunk 2]
```diff
@@ -218,21 +218,9 @@ export class MetamaskPage implements WalletPage {
...
-    });
-  }
-
-  async cancelSignTx(page: Page) {
-    await test.step('Cancel Sign TX', async () => {
-      await new WalletOperationPage(page).cancelSignTransaction();
-    });
-  }
-
   async rejectTx(page: Page) {
```
Comment (itaven) [inline packages/wallets/src/metamask/metamask.page.ts:221]: Should we remove wording wording reject->cancel?

File: packages/wallets/src/metamask/pages/walletOperations.page.ts (hunk 1 above)
Comment (jake4take) [reply packages/wallets/src/metamask/pages/walletOperations.page.ts]: renamed to "cancel"

File: packages/wallets/src/metamask/pages/walletOperations.page.ts [hunk 3]
```diff
@@ -60,22 +60,17 @@ export class WalletOperationPage {
...
-    await this.rejectButton.click();
-  }
-
-  async cancelSignTransaction() {
-    await this.cancelButton.click();
-  }
-
-  async signTransaction() {
-    await this.confirmButton.click();
+  async cancelTransaction() {
```
[the remark this answers is not in the document]
Comment (jake4take) [reply packages/wallets/src/metamask/pages/walletOperations.page.ts:68]: I've used the try catch cause for the unlocking tx in the MM cancel button got another `test-dataid`, But for other transactions the Cancel button locator is another
I think they will fix that `test-dataid` in the future

File: packages/wallets/src/metamask/metamask.page.ts (hunk 2 above)
Comment (jake4take) [reply packages/wallets/src/metamask/metamask.page.ts]: renamed all locators and functions (`reject` to `cancel`)

**Review by itaven — APPROVED** [review]

**Pull request merged**